README.md

```diff
@@ -12,10 +12,11 @@ kalliope install --git-url https://github.com/kalliope-project/kalliope_neuron_w
 ## Options
 
 | parameter | required | default | choices                     | comment                                                                                                                           |
-|-----------|----------|---------|-----------------------------|-----------------------------------------------------------------------------------------------------------------------------------|
-| language  | yes      |         | E.g: "fr", "en", "it", "es" | See the list of available language in the "Note" section                                                                          |
-| query     | yes      |         |                             | The wikipedia page you are looking for.  This parameter can be passed as an argument in the neuron from the order with {{ query}} |
+|-----------|----------|---------|-----------------------------|---------------------------------------------------------------------------------------------------------------------------------------|
+| language  | yes      |         | E.g: "fr", "en", "it", "es" | See the list of available language in the "Note" section                                                                    |
+| query     | yes      |         |                             | The wikipedia page you are looking for.  This parameter can be passed as an argument in the neuron from the order with {{ query}}  |
 | sentences | no       | 10      | Integer in range 1-10       | if set, return the first number of sentences(can be no greater than 10) specified in this parameter.                              |
+| titles    | no       | ~       | Integer                     | With this parameter you can limit the number of search result in case there are more then one                                     |
```

Comment:
maybe titles is not the right name for this parameter

Reply:
According to [wikipedia.summary](https://wikipedia.readthedocs.io/en/latest/code.html#wikipedia.summary) those are auto_suggest.
So, what about `auto_suggest` or `auto_suggest_limit` ?